tests/unix_listener.rs

```diff
@@ -1,8 +1,11 @@
-#![cfg(all(unix, feature = "os-poll", feature = "net"))]
+#![cfg(all(feature = "os-poll", feature = "net", any(unix, feature = "os-ext")))]
```

Comment:
Why do we need `os-ext` here? Removing `unix` enough?

```suggestion
#![cfg(all(feature = "os-poll", feature = "net"))]
```

Reply:
Ah good catch, previously I was exposing the blocking Windows UDS types under `os-ext` for Windows and so I'd added this to ensure we could access the Windows `std::net` types on Windows.